platform/security/claims/security-claims-attributequerycommon/src/main/java/org/codice/ddf/security/claims/attributequery/common/AttributeQueryClaimsHandler.java

```diff
@@ -281,11 +288,37 @@ private void auditRemoteConnection(URL wsdlURL) {
     return dispatch;
   }
 
-  public void setSignatureProperties(Object signatureProperties) {
+  public void setSignaturePropertiesLocation(String signaturePropertiesLocation) {
+    this.signaturePropertiesLocation =
+        PropertyResolver.resolveProperties(signaturePropertiesLocation);
+
+    try {
+      this.setSignatureProperties(
+          PropertyResolver.resolvePropertiesFromLocation(this.signaturePropertiesLocation));
+    } catch (IOException e) {
+      LOGGER.warn(
+          "Problem reading signature properties file {}.", this.signaturePropertiesLocation);
+    }
+  }
+
+  public void setEncryptionPropertiesLocation(String encryptionPropertiesLocation) {
+    this.encryptionPropertiesLocation =
+        PropertyResolver.resolveProperties(encryptionPropertiesLocation);
+
+    try {
+      this.setEncryptionProperties(
+          PropertyResolver.resolvePropertiesFromLocation(this.encryptionPropertiesLocation));
+    } catch (IOException e) {
+      LOGGER.warn(
+          "Problem reading encryption properties file {}.", this.encryptionPropertiesLocation, e);
+    }
+  }
+
+  public void setSignatureProperties(Properties signatureProperties) {
     this.signatureProperties = signatureProperties;
   }
 
-  public void setEncryptionProperties(Object encryptionProperties) {
+  public void setEncryptionProperties(Properties encryptionProperties) {
```

Comment:
✏️ Nit: Add `@VisibleForTesting` to `setSignatureProperties` and `setEncryptionProperties`.